README.md

```diff
@@ -103,7 +103,7 @@ Environment:
 |	MF_AGENT_MQTT_CLIENT_PK                | Location of client certificate key for MTLS                   | thing.key                         |
 
 Here `thing` is a Mainflux thing, and control channel from `channels` is used with `req` and `res` subtopic
-(i.e. app needs to PUB/SUB on `/channels/<channel_id>/messages/req` and `/channels/<channel_id>/messages/res`).
+(i.e. app needs to PUB/SUB on `/channels/<control_channel>/messages/req` and `/channels/<control_channel>/messages/res`).
```

Comment:
`control_channel_id` ?

Reply:
@drasko `control_channel` or `control_channel_id`?

Reply:
I can accept this, it is clear enough and short/simple enough.

Reply:
@drasko If it's an ID it's not clear. It could be a name.

Reply:
I agree, adding `_id` does not cost much and it is more precise.